packages/relay-compiler/bin/RelayCompilerMain.js

```diff
@@ -97,74 +116,67 @@ type LanguagePlugin = PluginInitializer | {default: PluginInitializer};
  * replaced with `__webpack_require__` when bundled using webpack, by using
  * `eval` to get it at runtime.
  */
-function getLanguagePlugin(language: string): PluginInterface {
+function getLanguagePlugin(
+  language: string | PluginInitializer,
+): PluginInterface {
   if (language === 'javascript') {
     return RelayLanguagePluginJavaScript();
   } else {
-    const pluginPath = path.resolve(process.cwd(), language);
-    const requirePath = fs.existsSync(pluginPath)
-      ? pluginPath
-      : `relay-compiler-language-${language}`;
-    try {
-      // eslint-disable-next-line no-eval
-      let languagePlugin: LanguagePlugin = eval('require')(requirePath);
-      if (languagePlugin.default) {
-        languagePlugin = languagePlugin.default;
+    let languagePlugin: LanguagePlugin;
+    if (typeof language === 'string') {
+      const pluginPath = path.resolve(process.cwd(), language);
```

Comment:
This module loading from a path could probably be deprecated with the ability to just pass a function using a `relay.config.js` file.

Reply:
yeah, let's do that. we can deprecate this in a follow-up and  delete in a later release